src/connection.cpp

```diff
@@ -116,7 +116,8 @@ void Looper::serve()
     {
         mg_mgr_poll(m_mgr.get(), m_copts.timer_poll_interval_ms);
         getTimerList().eval();
-        if(stopped()) break;
+
+        if(stopped() && canStop()) break;
```

Comment:
Actually, in this case, I'd like SIGTERM to **kill** the threads in the threadpool